### DISCUSSION TITLE
<Tab> completion dosn't work

### DISCUSSION THREAD

**Discussion by mattthhh** [CopilotC-Nvim/CopilotChat.nvim#412] [Q&A] [answered]
I'm using the original Copilot plugin from GitHub and it's working perfectly when I edit a file in a buffer. When I toggle the Copilot chat window and start writing, I can see the completion suggestion but the map <Tab> doesn't work. When I start the command `:Copilot status`, it tells me that it's online and enabled. I guess the mapping is disabled on the window, or the fallback mapping used by GH Copilot (`\t`) doesn't allow the completion in the chat window. Any ideas how to resolve this ? (without changing the mapping if possible)

**Comment by distributed-clouds** [comment, marked as answer]
I have the same issue but by removing the <tab> it is now working for me.

```
			mappings = {
				-- Use tab for completion
				complete = {
					detail = "Use @<Tab> or /<Tab> for options.",
					insert = "",
				},
```

**Reply by jamis** [reply]
In case anyone is as confused by this as I was (I'm not sure what was being "removed" in the above?), I was able to make this work by pasting the above snippet into the "opts" section of the copilot-chat plugin config. Using lazy, my config looks something like:

```lua
return {
  {
    "CopilotC-Nvim/CopilotChat.nvim",
    ...
    opts = {
      mappings = {
        -- Use tab for completion
        complete = {
          detail = "Use @<Tab> or /<Tab> for options.",
          insert = "",
        },
      },
    },
    ...
  }
}
